### PR TITLE
New ERT: The Inter-Dimensional Police Department

### PR DESCRIPTION
**Pull request by Kapu1178**
## About The Pull Request
I like nuclear throne, and apparently @PestoVerde322 likes it enough to sprite these hardsuits for me.

Adds the I.D.P.D ERT, an ERT that spawns from a portal at a random on-station location. Their power level is intended to be closer to Code Red.

## Why It's Good For The Game
A new, more unique ERT option that will get used one time.

## Changelog
:cl: Kapu1178, PestoVerde322
add: Inter-Dimensional Police Department ERT
imageadd: I.D.P.D hardsuits
/:cl:

### CODE REVIEW

**Comment by Kapu1178** [comment]
https://cdn.discordapp.com/attachments/484170915253321734/905632585185628160/idpdert.mp4

File: code/modules/admin/verbs/one_click_antag.dm [hunk 1]
```diff
@@ -430,3 +432,60 @@
 /datum/admins/proc/makeRevenant()
 	new /datum/round_event/ghost_role/revenant(TRUE, TRUE)
 	return 1
+
+/datum/admins/proc/attemptMakeIDPD(var/list/mob/dead/observer/candidates, var/num_agents, var/datum/ert/ertemplate, var/datum/team/ert/ert_team, var/datum/objective/missionobj) //They don't start at centcom (fuck you linter)
```
Comment (Crossedfall) [inline code/modules/admin/verbs/one_click_antag.dm]:
```suggestion
/datum/admins/proc/attemptMakeIDPD(var/list/mob/dead/observer/candidates, var/num_agents, var/datum/ert/ertemplate, var/datum/team/ert/ert_team, var/datum/objective/missionobj) //They don't start at centcom (thank you linter)
```

Either remove it or change it.

**Review by Crossedfall — CHANGES_REQUESTED** [review]

File: code/modules/admin/verbs/one_click_antag.dm (hunk 1 above)
Comment (Kapu1178) [reply code/modules/admin/verbs/one_click_antag.dm]: grumble grumble

**Comment by Crossedfall** [comment]
While the sprites look really nice... Based on my quick search, these sprites are incredibly similar to the source material. To the point where I'd say they might need to be changed. Inspired by is fine, provided that they're still original, direct rip isn't as fine.

**Comment by Kapu1178** [comment]
> 
> 
> While the sprites look really nice... Based on my quick search, these sprites are incredibly similar to the source material. To the point where I'd say they might need to be changed. Inspired by is fine, provided that they're still original, direct rip isn't as fine.

the gag is they are coming from their dimension to ours to "fix" troublemakers thooooooooo

**Comment by Crossedfall** [comment]
> > While the sprites look really nice... Based on my quick search, these sprites are incredibly similar to the source material. To the point where I'd say they might need to be changed. Inspired by is fine, provided that they're still original, direct rip isn't as fine.
> 
> the gag is they are coming from their dimension to ours to "fix" troublemakers thooooooooo

Unless you got the OK from Vlambeer, I'm going to have to say no.

Closing due to the sprites issue. Once that's resolved, let me know and I'll be more than happy to re-open this.

**Pull request closed without merging**